source/pages/Tutorial/dev-client-winrt.rst

```diff
@@ -1,193 +1,200 @@
+.. important::
+   All commands must be executed in Developer PowerShell for Visual Studio 2022.
 
-6. Save and close the file.
+Packing and Signing the Modified Client
+-----------------------------------------
 
-Next, pack and sign the modified client with these commands:
+Pack and sign the modified APPX in Developer PowerShell for Visual Studio 2022:
 
-::
+.. code-block:: console
 
-    makeappx pack /d BraveFrontierAppxClient /p BraveFrontierPatched.appx
-    SignTool sign /a /v /fd SHA256 /f MyKey.pfx /p "<Your Password>" BraveFrontierPatched.appx
+   makeappx pack /d C:\BF\BraveFrontierAppxClient /p C:\BF\BraveFrontierPatched.appx
+   SignTool sign /a /v /fd SHA256 /f C:\BF\MyKey.pfx /p "YourStrongPassword" C:\BF\BraveFrontierPatched.appx
 
 .. note::
-
-   Replace ``<Your Password>`` with the password used when exporting the certificate.
+   Ensure the password matches the one used during certificate export.
 
 Running the Game
-----------------
+------------------
 
-Install the newly generated ``BraveFrontierPatched.appx`` file and launch the client. If all steps were followed correctly, a console window should appear alongside the game client, as shown below:
+Install the patched client:
 
-.. image::
-    ../../images/dev-client-winrt/bf_appx_patched.png
+.. code-block:: powershell
 
-.. warning::
+   Add-AppxPackage C:\BF\BraveFrontierPatched.appx
 
-   If no console appears, check the following:
-   
-   - Ensure the patched ``libcurl.dll`` was correctly installed.
+Enable loopback for local server communication:
 
-   - Verify you did not use the ``deploy`` preset, as it is not supported in this build.
+#. Download the Enable Loopback Utility: https://telerik-fiddler.s3.amazonaws.com/fiddler/addons/enableloopbackutility.exe
+#. Run the utility and select "Brave Frontier".
+#. Check "Enable loopback" and click "Save Changes".
+#. If an error occurs, enable Device Portal in Settings > Update & Security > For developers and disable "Restrict to loopback connections only".
 
-   - Confirm Developer Mode is enabled on your Windows PC.
+.. image:: ../../images/dev-client-winrt/loopback_win.png
+   :alt: Loopback Utility Configuration
 
-Connecting to the Server
-~~~~~~~~~~~~~~~~~~~~~~~~
+Launch the game by searching "Brave Frontier" in the Start menu. A console window should appear alongside the client.
 
-Due to a default limitation in UWP apps, they cannot communicate with localhost, preventing the game from connecting to the server. To resolve this:
+.. image:: ../../images/dev-client-winrt/bf_appx_patched.png
+   :alt: Running the Patched Game Client
 
-1. Download the `Enable Loopback Utility <https://telerik-fiddler.s3.amazonaws.com/fiddler/addons/enableloopbackutility.exe>`_.
-2. Run the utility and select the Brave Frontier application.
-3. Configure it as shown below, then click ``Save Changes`` and restart the game:
+.. warning::
+   If no console appears, verify the following:
```

Comment:
If your intended result is this:
<img width="471" height="209" alt="msedge_yOeTeOUQwW" src="https://github.com/user-attachments/assets/49e008e1-d7c0-47bc-8f2b-60358e0188b8" />

You need to add a space after following:
```

.. warning::
   If no console appears, verify the following:
   
   - The patched ``libcurl.dll`` was correctly installed.
   - Developer Mode is enabled on your Windows PC.
```